### ISSUE TITLE
People Using Tor Proxies

### ISSUE THREAD

**Issue by tylerhyperHD** [TotalFreedom/TotalFreedomMod#1739]
It has been an increasing issue in which spammers are using Tor, a popular proxy and browser, to spam, flood, and crash the server. I found someone on github who created a proxy blocker that was able to get rid of the ENTIRE Tor network:
http://github.com/socram8888/ProxyKicker

Something like this would be insanely useful to get rid of people using these types of proxies to bypass bans and to create chaos on the server. I highly recommend that we start working on a way to fix these problems.

**Comment by Wild1145** [comment]
I don't think we need to be blindly blocking tens of thousands of IP's on a wim...

**Comment by tylerhyperHD** [comment]
Well this is why a VPS would really improve this, due to the fact it would allow more fluid connection blocking. The issue at hand is the permban list right now is already way too long, and I don't know how there is any way we could solve that.

**Comment by AssassinDude** [comment]
just sayying, it wont help unless u take cracked mode off and make it only a premuim but u guys wont do that so people like me take advantage of that and get an actual ip changer no not a proxy, (proxy sucks anyway) an actual ip changer and come on when the server is cracked or get a alt account off of MCLeaks and terrorize the server

**Comment by Wild1145** [comment]
We could solve the permban list issue by not permbanning people for minor offences like we currently do.

As I've said on the thread, there is nothing in the rules against the use of Proxies and VPN's, so we will not block them.
